numpy/_core/include/numpy/dtype_api.h

```diff
@@ -187,6 +196,7 @@ typedef NPY_CASTING (resolve_descriptors_with_scalars_function)(
         npy_intp *view_offset);
 
 
+
```

Comment:
Should we give the other typedefs in this file names like `PyArrayMethod_ResolveDescriptorsFunction`?